dev-config.js

```diff
@@ -8,6 +12,9 @@ module.exports = function createDevConfig(serviceWorker, favPath) {
 		plugins.push("new FaviconsWebpackPlugin('"+ favPath +"')");
 	}
 
+	let fd = fs.openSync('manifest.json', 'w');
+	fs.writeFile(fd, createManifest(manifestDetails), 'utf8');
```

Comment:
[`writeFile`](https://nodejs.org/api/fs.html#fs_fs_writefile_file_data_options_callback) is an asynchronous call, there might be issues if we write this file at other places without waiting for the callback here.

I believe [`createWriteStream`](https://nodejs.org/api/fs.html#fs_fs_createwritestream_path_options) is synchronous. Can we use it here?

Also, `'utf8'` is default encoding option. We can clean this up.

Reply:
probably `writeFileSync`